packages/bot-web-ui/src/components/bot-footer-extensions/bot-footer-extensions.tsx

```diff
@@ -11,22 +10,16 @@ type TPopulateFooterExtensionsProps = {
 };
 
 type TBotFooterExtentionsProps = {
-    active_tab: string;
     populateFooterExtensions: (footer_extensions: TPopulateFooterExtensionsProps[]) => void;
-    setActiveTab: (tab_title: string) => void;
 };
 
-const BotFooterExtensions = ({ active_tab, populateFooterExtensions, setActiveTab }: TBotFooterExtentionsProps) => {
+const BotFooterExtensions = ({ populateFooterExtensions }: TBotFooterExtentionsProps) => {
     React.useEffect(() => populateFooter());
     React.useEffect(() => () => populateFooterExtensions([]), [populateFooterExtensions]);
```

Comment:
Nit, because I know it's not your change, but can we add a dependency array for the `useEffect`.